### DISCUSSION TITLE
How to enable HTTP input plugin in kubernetes cluster

### DISCUSSION THREAD

**Discussion by pandagodyyy** [fluent/fluent-bit#5411] [Q&A]
I expect enable http input plugin in the K8s so that the microservices  can send custom events to fluent-bit.
I installed fluent-bit with helm way, and configured as below:
values:
    config:
            inputs: |
              [INPUT]
                  Name http
                  Host 0.0.0.0
                  Port 8888

I saw the log of fluent-bit and find log [2022/05/05 06:32:55] [ info] [input:http:http.0] listening on 0.0.0.0:8888
I think the input is initialized but I do not see 8888 endpoint created, besides http server 2020 for monitoring.

Does someone knows how I can make it work

**Comment by tanmay-bhat** [comment]
Hey @pandagodyyy , can you cgeck if port 8888 is exposed via the service? you can check that with the command : 
`kubectl get svc <fluent-bit-service-name>`

Also I belive you need to enable that extra port using the below config in your helm values file. Its not enabled by default. 
```
extraPorts: []
#   - port: 5170
#     containerPort: 5170
#     protocol: TCP
#     name: tcp
#     nodePort: 30517
```
